examples/basic.built.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset='utf-8'>
     <title>Replace me with a real title</title>
-    <script src="../builds/respec-w3c-common.js" class='remove'></script>
+    <script src="../builds/respec-w3c.js" class='remove'></script>
```

Comment:
We have several tests that uses respec-w3c-common, those should be fixed as well.

Reply:
I'll change those too.